iceoryx_posh/test/moduletests/test_roudi_process.cpp

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) 2020 by Robert Bosch GmbH. All rights reserved.
```

Comment:
I guess this was really created last year :)

Reply:
Done

Reply:
@chiranjeevimaddi why did you changed the copyright year to 2021? The year in which the file was created needs to be filled in there, not year it was changed the last time.

Reply:
@dkroenke this was a bit misunderstood from my side, i though @elBoberido was pointing to change, but with your comment i understood it was my fault. Do you want me to change it back to 2020?

Reply:
@chiranjeevimaddi sorry, it was just fun because we constantly do this wrong by c&p and this time it seemed to be correct, even if we already are in 2021. I'm fine with keeping it as it currently is, since the file lands in 2021 in master.